dl.py

```diff
@@ -1,13 +1,36 @@
 import sys
 from multiprocessing import Process
 from pytube import YouTube
+import ffmpeg
 
 
 def dl_vid(url):
     yt = YouTube(url)
-    print('downloading: ' + yt.title)
-    yt.streams.filter(progressive=True, file_extension='mp4').order_by('resolution').desc().first().download() 
-    print('finished: ' + yt.title)
+    title = yt.title
+    print('Downloading: ' + title)
+
+    # Check if text contains # symbol, if so replace as ffmpeg doesn't like #
+    if title.__contains__("#"):
+        title = title.replace("#", "")
+        print("AFTER: " + title)
+
+    # trim to replace whitespace with underscore
+    title_trim = "_".join(title.split())
+
+    yt.streams.filter(progressive=True, file_extension='mp4')\
+        .order_by('resolution')\
+        .desc()\
+        .first()\
+        .download(filename=title_trim)
+
+    convert(title_trim+".mp4", title_trim+".mp3")
+    print('Finished downloading: ' + yt.title)
+
+
+def convert(name_vid, name_sound):
+    print("name_vid:" + name_vid + "\n")
+    audio = ffmpeg.input(name_vid).audio.filter("aecho", 0.8, 0.9, 1000, 0.3)
```

Comment:
remove the filter applying an echo to the audio and we're good `.filter("aecho", 0.8, 0.9, 1000, 0.3)`